src/main/java/magicbees/main/Config.java

```diff
@@ -247,7 +252,7 @@ private void doModuleConfigs() {
         p = configuration.get(CATEGORY_MODULES, "ArsMagica", true);
         arsMagicaActive = p.getBoolean();
 
-        p = configuration.get(CATEGORY_MODULES, "BloodMagic", true);
+        p = configuration.get(CATEGORY_MODULES, "AWWayofTime", true);
```

Comment:
Why are you changing the config variable name? Please revert

Reply:
Was trying to figure out where the loading issue was